modules/kubecf/upgrade.sh

```diff
@@ -7,7 +7,11 @@
 info "Upgrading CFO…"
 helm list -A
 
-helm_upgrade cf-operator cf-operator/ \
+OPERATOR_DIR=cf-operator
+if [ -d quarks ]; then
```

Comment:
Do we need to check that it's the right shape?
```suggestion
if [ -f quarks/Chart.yaml ]; then
```

Reply:
> Do we need to check that it's the right shape?

I don't think it makes any difference, so I'll just merge in your suggested change :)